operations/agecategories.md

```diff
@@ -38,6 +42,7 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
 | `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of [Services](services.md#service) where the age category belong to. |
 | `AgeCategoryIds` | array of string | optional, max 1000 items | Unique identifiers of [Age categories](#age-category). |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which [Rule](#rule) was updated. |
```

Comment:
```suggestion
| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which [Rule](#rule) was last updated. |
```
We should be consistent here, but I think it wouldn't hurt to add word 'last' as that's reality, it is the last update.

Reply:
This would be nice to change across whole documentation, not just in this PR.